applications/FluidDynamicsApplication/custom_conditions/navier_stokes_wall_condition.cpp

```diff
@@ -363,6 +369,11 @@ const ConditionDataStruct& data)
         Vector rhs_gauss_behr = ZeroVector(TNumNodes*LocalSize);
```

Comment:
What is pointed above also applies for the RHS.

Reply:
I think one should be careful here. `rhs_gauss` is initialized as a zero vector, but it also carries the contributions from `ComputeRHSNeumannContribution(rhs_gauss, data)` and (possibly) also the outlet contribution. In the current implementation, those would be overwritten.
Thus, the following suggestion: Reimplementation of the function, such that they only ADD to an already given argument (meaning "+=" instead of "=").

Reply:
This is the idea, to use += instead of =

Reply:
Thank you. Done.